inc/group_user.class.php

```diff
@@ -309,7 +309,8 @@ class='submit'>";
    **/
    private static function showAddUserForm(Group $group, $used_ids, $entityrestrict, $crit) {
       $rand = mt_rand();
-      $res  = User::getSqlSearchResult(true, "all", $entityrestrict, 0, $used_ids);
+      $res  = User::getSqlSearchResult(true, "all", $entityrestrict, 0, $used_ids, '', 0, -1, 0, 1);
```

Comment:
Sounds like there is one extra argument (see getSqlSearchResult method signature).

Reply:
yes the new one =)

![image](https://user-images.githubusercontent.com/7335054/116086519-32ab5700-a6a0-11eb-9de4-9c87e9b1dcb3.png)

Reply:
Ok... I'm tired :D